cosmos/providers/dbt/core/profiles/glue/aws_role_arn.py

```diff
@@ -0,0 +1,52 @@
+"Maps Aws connections to dbt profiles."
+from __future__ import annotations
+
+from logging import getLogger
+from typing import Any
+
+from ..base import BaseProfileMapping
+
+logger = getLogger(__name__)
+
+
+class AWSRoleARNProfileMapping(BaseProfileMapping):
+    """
+    Maps Airflow Aws connections to Glue dbt profiles.
+
+    https://docs.getdbt.com/docs/core/connect-data-platform/glue-setup
+    https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/aws.html
+    https://github.com/aws-samples/dbt-glue
+    """
+
+    airflow_connection_type: str = "aws"
```

Comment:
I wonder if we need to let the user specify which dbt profile type they want - right now we assume the mapping based on whether or not the required fields are there, but that may not be sufficient if there are cases where there's on airflow connection -> many dbt profiles

Reply:
I agree, @jlaneve, AWS connection is just one of these scenarios.

I believe it is important addressing this concern as part of this PR, otherwise, in future, we'll likely have to introduce a breaking change to handle this scenario.